aws-rds-globalcluster/src/test/java/software/amazon/rds/globalcluster/ReadHandlerTest.java

```diff
@@ -32,12 +41,6 @@
 
     private ReadHandler handler;
 
-    @AfterEach
-    public void post_execute() {
```

Comment:
Do we need to add this to all tests if we remove it from here?

Reply:
I think best to keep verify in the test, then it will be apparent which test failed.

Reply:
but you should add it to each test, with the current change we don't have the verify

Reply:
that verifies the interactions with the mocks, the new tests didn't use the mocks. so actually verifying at the end of tests that don't use mocks would fails those tests.